From 9aecfae9087ea4247a1a41d455f0f60e3a68e6bc Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 21 Jun 2024 14:27:31 +0200
Subject: [PATCH 0931/1695] drivers: stm32_exti: use generic CIDCFGR macros

Use _CIDCFGR_SCID_SHIFT instead of SCID_SHIFT.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 99a70309856ad70dc9535a24864ef0c96c0d0c6c drivers: stm32_exti: use generic CIDCFGR macros]
Change-Id: Idef3d41eb88cda23cb441b21fc99d68e8760554f
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389465
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
---
 core/drivers/stm32_exti.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/drivers/stm32_exti.c b/core/drivers/stm32_exti.c
index cc616cf4b..811d7adec 100644
--- a/core/drivers/stm32_exti.c
+++ b/core/drivers/stm32_exti.c
@@ -319,7 +319,8 @@ static void stm32_exti_rif_parse_dt(struct stm32_exti_pdata *exti,
 		if (c_cid > stm32_exti_maxcid(exti))
 			panic("CID out of range");
 
-		exti->c_cids[pos - 1] = (c_cid << SCID_SHIFT) | _CIDCFGR_CFEN;
+		exti->c_cids[pos - 1] = (c_cid << _CIDCFGR_SCID_SHIFT) |
+					_CIDCFGR_CFEN;
 	}
 }
 
-- 
2.47.2

